docs/multicluster/architecture.md

```diff
@@ -201,3 +200,27 @@ be created as a ResourceExport in the leader cluster, and the resource
 export/import pipeline will ensure member clusters receive this ACNP spec
 to be replicated. Each member cluster's Multi-cluster Controller will then
 create an ACNP in their respective clusters.
+
+## Antrea Traffic Modes
+
+Multi-cluster Gateway supports all of `encap`, `noEncap`, `hybrid`, and
+`networkPolicyOnly` modes. In all supported modes, the cross-cluster traffic
+is routed by Multi-cluster Gateways of member clusters, and the traffic goes
+through Antrea overlay tunnels between Gateways. In `noEncap`, `hybrid`, and
+`networkPolicyOnly` modes, even when in-cluster Pod traffic does not go through
+tunnels, antrea-agent still creates tunnels between the Gateway Node and other
+Nodes, and routes cross-cluster traffic to reach the Gateway through the tunnels.
+Specially for `networkPolicyOnly` mode, Antrea only handles multi-cluster traffic
+routing, while the primary CNI takes care of in-cluster traffic routing.
+
+### Antrea Multi-cluster with NetworkPolicy Only Mode
```

Comment:
This section can be removed? It just repeats the contents in user guide, and needs not to be in the architecture doc. For policyOnly mode CNI, we can let users refer to the policy-only.md.

Reply:
Removed